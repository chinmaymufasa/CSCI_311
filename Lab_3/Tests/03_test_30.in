2
95
5
2
34
5
2
61
5
2
34
5
2
106
5
2
8
5
2
170
5
2
163
5
2
23
5
2
149
5
2
183
5
2
56
5
2
92
5
2
162
5
2
64
5
2
119
5
2
138
5
2
126
5
2
163
5
2
19
5
1
20
-1