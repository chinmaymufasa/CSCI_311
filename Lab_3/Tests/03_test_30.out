95 
34 95 
34 61 95 
34 34 61 95 
34 34 61 95 106 
8 34 34 61 95 106 
8 34 34 61 95 106 170 
8 34 34 61 95 106 163 170 
8 23 34 34 61 95 106 163 170 
8 23 34 34 61 95 106 149 163 170 
8 23 34 34 61 95 106 149 163 170 183 
8 23 34 34 56 61 95 106 149 163 170 183 
8 23 34 34 56 61 92 95 106 149 163 170 183 
8 23 34 34 56 61 92 95 106 149 162 163 170 183 
8 23 34 34 56 61 64 92 95 106 149 162 163 170 183 
8 23 34 34 56 61 64 92 95 106 119 149 162 163 170 183 
8 23 34 34 56 61 64 92 95 106 119 138 149 162 163 170 183 
8 23 34 34 56 61 64 92 95 106 119 126 138 149 162 163 170 183 
8 23 34 34 56 61 64 92 95 106 119 126 138 149 162 163 163 170 183 
8 19 23 34 34 56 61 64 92 95 106 119 126 138 149 162 163 163 170 183 
Not found
